syntax = "proto3";

package course_score_query;

import "course_score_query/model.proto";

service CourseScoreQuery {
    rpc FindCourseByName(FindCourseByNameRequest) returns (FindCourseByNameResponse);
    rpc FetchLayeredScoreDetail(FetchLayeredScoreDetailRequest) returns (FetchLayeredScoreDetailResponse);
}